tests/components/ct-vue-vite/src/notation-vue.spec.ts

```diff
@@ -128,3 +125,12 @@ test('get textContent of the empty template', async ({ mount }) => {
   expect(await component.textContent()).toBe('');
   await expect(component).toHaveText('');
 });
+
+test('render app and navigate to a page', async ({ page, mount }) => {
+  const component = await mount(App);
+  await expect(component.getByRole('main')).toHaveText('Login');
+  await expect(page).toHaveURL('/playwright/index.html/');
```

Comment:
I can't get rid of `/playwright/index.html/` until https://github.com/microsoft/playwright/pull/17767 is fixed